e2e/__tests__/findRelatedFiles.test.ts

```diff
@@ -116,6 +116,56 @@ describe('--findRelatedTests flag', () => {
     expect(stderr).toMatch(summaryMsg);
   });
 
+  test('runs tests related to filename with a custom dependency extractor written in ESM', () => {
```

Comment:
@SimenB I am not sure if this is a valid e2e test of ESM dependency extractor. I just found a commonjs version above and did the same against ESM.